.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
We could generate these files for PRs too.

Reply:
yeah i believe this will be done when we start automation PRs etc